chore: add a problem and a solution for sync .env file (#1687)
freeCodeCamp/chapter: CONTRIBUTING.md

  > *Invalid'prisma_1.prisma.chapters.findMany()* </br>

  **Solution:** The [database needs to be initialized](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md#initializing-the-database). Run `npm run db:reset` to clear and re-create the database tables.

>   * **Problem:**  I have come back after a long period of time, and chapter isn't working. </br>
>     **Solution:** In chapter development, `.env.example` has changed quite a bit. Be sure to sync up your .env with .env.example or things won't work until you do.

Reviewer: ```suggestion
* **Problem:**  I have come back after a long period of time, and chapter isn't working. </br>
  **Solution:** In chapter development, `.env.example` has changed quite a bit. Be sure to sync up your .env with .env.example or things won't work until you do.
```
Indentation makes it appear as a subsection of the previous solution.

Wording could be slightly more pointing this as a _possible_ solution, but I don't have any specific idea how to change that.

Author: We can change `or things won't work until you do`, to something like `so the codebase could run smoothly`.

```suggestion
  **Solution:** In chapter development, .env.example has changed quite a bit. Be sure to sync up your .env with .env.example so the codebase could run smoothly.
```

Reviewer 2: It's more that we don't know if the example has changed or if that's the problem they're facing.

How about this: "As we develop Chapter it is often necessary to change the environment variables we use. In case your .env is out of date, try replacing it with .env.example"?
